### ISSUE TITLE
[Documentation] Tooltips formatter parameters for 2 handles is not intuitive

### ISSUE THREAD

**Issue by zsitro** [leongersen/noUiSlider#1101]
Hi, noUiSlider rocks! 

The problem I had was that I tried to set up a custom formatter with a `function(v){...}` for two handles and I tried all kinds of variations, like:

```
tooltips: [{
  to: function(v){ return ~~v * vm.display_multiplier; },
  from: function(v){ return ~~v * vm.display_multiplier; }
}]
```

or 

```
tooltips: [
  function(v){ return ~~v * vm.display_multiplier;}, 
  function(v){ return ~~v * vm.display_multiplier;}
]
```

but none worked of course and I had to check the source code to figure the solution that worked:

```
tooltips: [
  {
    to: function(v){ return ~~v * vm.display_multiplier; },
  },{
    to: function(v){ return ~~v * vm.display_multiplier; }
  }
],
```

Which I found very unintuitive. An example in the documentation could have saved me some time to figure this out.

What do you think?

**Comment by leongersen** [comment]
I'm always more than happy to improve documentation. There's currently two sections detailing tooltip formatting:

https://refreshless.com/nouislider/slider-options/#section-tooltips
https://refreshless.com/nouislider/number-formatting/#section-tooltips

They both use the same example. Would it help (or have helped, rather) if more examples were presented on either of the locations?

**Comment by zsitro** [comment]
Thanks for your feedback. Based on the examples in the documentation I couldn't set up a working formatter.

If I follow the examples I get this:
```
tooltips: [
  function(v){ return ~~v * vm.display_multiplier;}, 
  function(v){ return ~~v * vm.display_multiplier;}
]
```

Which resulted in this error:
https://github.com/leongersen/noUiSlider/blob/master/src/nouislider.js#L929

**Comment by leongersen** [comment]
Ok, that helps. I'll add some clarification to the documentation. Thanks for reaching out!

**Comment by d-miles** [comment]
Hi @leongersen,

I figured I would just respond here rather than creating a new issue as it is directly related to the documentation change made for this issue. The documentation change suggests the following to apply a default formatter for **all handles**:

```
tooltips: {
        to: function (v) {
            ...
        },
        from: function (v) {
            ...
        }
}
```

However, when doing this I receive the following error:

```
Error: noUiSlider: must pass a formatter for all handles.
```

I still must provide an array of tooltip formatters for all handles, rather than the suggested single object formatter to apply a single formatter to all handles. Is this just a bug in the documentation or in the tooltips itself? There doesn't seem to be a way that I can find to set a default formatter for all handlers.

Please let me know if you'd prefer that a new issue is created to address this! :)

**Comment by leongersen** [comment]
@d-miles Thanks for following up on this! The documented option was definitely the intent, but this wasn't properly implemented. I've fixed this in noUiSlider 15.1.1. 

@zsitro My apologies for not actually checking that this feature worked correctly before updating the documentation!

Thanks for contributing!

**Comment by d-miles** [comment]
Thanks for such a quick response @leongersen! I've updated and confirmed this is working as expected! 

Thank you! Keep up the good work ;)
